.github/workflows/cleanup-storybook-preview.yml

```
@@ -0,0 +1,30 @@
+
+jobs:
+  cleanup-s3-preview:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: 🔐 Configure AWS credentials using OIDC
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/${{ secrets.AWS_DEPLOY_ROLE }}
+          aws-region: us-west-2
+          audience: sts.amazonaws.com 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)